app/src/main/java/ba/sum/fpmoz/mim/ui/adapters/SubjectAdapter.java

```diff
@@ -0,0 +1,83 @@
+package ba.sum.fpmoz.mim.ui.adapters;
+
+import android.content.Intent;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.firebase.ui.database.FirebaseRecyclerAdapter;
+import com.firebase.ui.database.FirebaseRecyclerOptions;
+
+import ba.sum.fpmoz.mim.R;
+import ba.sum.fpmoz.mim.SubjectAdminEditActivity;
+import ba.sum.fpmoz.mim.model.Subject;
+
+public class SubjectAdapter extends FirebaseRecyclerAdapter<Subject,SubjectAdapter.SubjectViewHolder> {
+    public SubjectAdapter(@NonNull FirebaseRecyclerOptions<Subject> options){
+        super(options);
+    }
+
+    @Override
+    protected void onBindViewHolder(@NonNull SubjectAdapter.SubjectViewHolder holder, int position, @NonNull Subject model) {
+        holder.subjectName.setText(model.getName());
+    }
+
+    @NonNull
+    @Override
+    public SubjectViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
+        View view= LayoutInflater.from(parent.getContext()).inflate(R.layout.viewholder_subject_admin,parent,false);
+        SubjectViewHolder viewHolder=new SubjectAdapter.SubjectViewHolder(view);
+        viewHolder.setOnClickListener(new Adapter.ClickListener() {
+            @Override
+            public void OnClickListener(View v, int position) {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.